server/config.ts

```diff
@@ -86,7 +86,6 @@ export default {
   app: {
     production,
     mailTo: process.env.MAIL_TO || 'feedback@digital.justice.gov.uk',
-    url: process.env.CHECK_MY_DIARY_URL || `http://localhost:${process.env.PORT || 3000}`,
```

Comment:
this wasn't now used after copying latest template changes - started to use `config.domain` instead